### PR TITLE
more /tg/station ports - tools! logging! some other fun stuff!

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
ports tgstation/tgstation#40046, tgstation/tgstation#41695, tgstation/tgstation#46726. mainly 40046. PLEASE read those changelogs i am not copypasting all of that over
## Why It's Good For The Game
40046: printable stuff also the Tool Designs category is really good
41695: logging good
46726: feature parity
## Changelog
🆑 Denton, ported by Hatterhat
add: Added new available items to all departmental protolathes. Keep an eye out for the new "Basic Tools" and "Basic Security Equipment" research nodes.
tweak: Made the AI surveillance upgrade available through the "Illegal Technology" research node.
tweak: Created a "Tool Designs" lathe subsection and moved most tools into it.
balance: Made defibrillators easier to print by moving them to the basic biotechnology research node.
fix: Health analyzers are now properly printable once researched.
tweak: AIs that get upgraded with a combat software upgrade now see a reminder that this doesn't change their laws or antagonist status.
admin: Added logging and an admin follow message to AI combat/surveillance software upgrades.
balance: The AI combat software upgrade now adds missing malf AI abilities, like being able to emag borgs.
/🆑

### CODE REVIEW

**Comment by MrJWhit** [comment]
I'm not completely positive on a spray bottle easily mass producible by medical, but everything else looks great.

**Comment by Hatterhat** [comment]
spray bottle is a service protolathe item, medspray is for synthflesh n stuff

**Comment by Owai-Seek** [comment]
The thing about spray bottles that you dont even need the service one, you can ask botany for pest spray and it functions identically to the spray bottle.

What -does- concern me is PRINTABLE, BEARTRAPS.

File: code/game/objects/items/robot/ai_upgrades.dm [hunk 1]
```diff
@@ -9,34 +9,44 @@
...
+		message_admins("[ADMIN_LOOKUPFLW(user)] has upgraded [ADMIN_LOOKUPFLW(AI)] with a [src].")
 	to_chat(user, "<span class='notice'>You upgrade [AI]. [src] is consumed in the process.</span>")
 	qdel(src)
 
 
 //Lipreading
 /obj/item/surveillance_upgrade
 	name = "surveillance software upgrade"
-	desc = "A software package that will allow an artificial intelligence to 'hear' from its cameras via lip reading."
+	desc = "An illegal software package that will allow an artificial intelligence to 'hear' from its cameras via lip reading and hidden microphones."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/robot/ai_upgrades.dm]: This is not illegal tho

**Comment by LetterN** [comment]
finally, science can print multitools

**Comment by Trilbyspaceclone** [comment]
Ive seen this before... #7822

**Comment by GrayRachnid** [comment]
Trilby, you closed that yourself, not a maintainer denying it.

**Comment by r4d6** [comment]
Once https://github.com/Citadel-Station-13/Citadel-Station-13/pull/10416/ is merged, could you update this PR to change the RCD, RPD & Upgrade Disks into the Tool category please? Since it is currently being test-merged I can't change it to need yours.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**